src/core/sagas/utils.js

```diff
@@ -1,9 +1,7 @@
-/* @flow */
```

Comment:
Sorry if this was giving you grief. I think it's important to keep Flow where we have it, especially since the problem I mentioned above (possible missing params in your action creator) are protected nicely by Flow. Here is a patch to put it back for you:

```diff
diff --git a/src/core/sagas/utils.js b/src/core/sagas/utils.js
index 65e66069..f35ab181 100644
--- a/src/core/sagas/utils.js
+++ b/src/core/sagas/utils.js
@@ -1,7 +1,12 @@
+/* @flow */
 import { ErrorHandler } from 'core/errorHandler';
 import defaultLog from 'core/logger';
+import type { ApiStateType } from 'core/reducers/api';
 
-export function createErrorHandler(id, { log = defaultLog } = {}) {
+export function createErrorHandler(
+  id: string,
+  { log = defaultLog }: {| log: typeof defaultLog |} = {}
+): typeof ErrorHandler {
   return new ErrorHandler({
     id,
     // Make sure the dispatch() method can't be used. A saga will yield
@@ -12,11 +17,11 @@ export function createErrorHandler(id, { log = defaultLog } = {}) {
 }
 
 // Convenience function to extract API info.
-export function getApi(state) {
+export function getApi(state: {| api: ApiStateType |}): ApiStateType {
   return state.api;
 }
 
 // Convenience function to extract authentication info.
-export function getAuth(state) {
+export function getAuth(state: Object) {
   return state.auth;
 }
```